pkg/lib/serverTime.js

```diff
@@ -321,13 +309,89 @@
                     console.warn("failed to load time servers", error);
                     return result;
                 });
-    };
+    }
+
+    function set_custom_ntp_timesyncd(config) {
+        const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
+                                                    { superuser: "try" });
 
-    self.set_custom_ntp = function (servers, enabled) {
-        const text = `# This file is automatically generated by Cockpit\n\n[Time]\n${enabled ? "" : "#"}NTP=${servers.join(" ")}\n`;
+        const text = `# This file is automatically generated by Cockpit\n\n[Time]\n${config.enabled ? "" : "#"}NTP=${config.servers.join(" ")}\n`;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18298-20230207-124146-95ee9c44-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)